src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,13 @@
-export const TodoList = () => {};
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: TodoList[],
```

Comment:
The type of `todos` prop is `TodoList[]`, but it should be `TodoInfo[]`. Consider changing the type of `todos` prop to `TodoInfo[]`. 🤔